airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -29,31 +29,42 @@
         "type": "string",
         "order": 2
       },
+      "schemas": {
+        "title": "Schemas",
+        "description": "The list of schemas to sync from. Defaults to user. Case sensitive.",
```

Comment:
Wdyt about adding info about comma separator if we use more than 1 schema, or it is obvious?

Reply:
Good question. Please check the first screenshot. The UI represents it as different items but sends it to the backend as an array. So no explicit separators are required